docs/src/rsh/errors/index.md

```diff
@@ -1734,8 +1734,7 @@ const s = Struct([["x", UInt], ["y", UInt], ["x2", UInt]]);
 ## {#RE0098} RE0098
 
 This error indicates that you are attempting to export a name that the
-Reach backend already produces. For example, the names provided in `{!rsh} Participant`,
-`{!rsh} ParticipantClass`, and `{!rsh} View` will be exported by the Reach backend.
+Reach backend already produces. For example, the names provided in `{!rsh} Participant`, `{!rsh} ParticipantClass`, `{!rsh} API`, and `{!rsh} View` will be exported by the Reach backend.
```

Comment:
This is fine, but also this is an example where the newlines should match the semantics.